frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/dev-console-create-vm-form.tsx

```diff
@@ -0,0 +1,239 @@
+import * as React from 'react';
+import { TFunction } from 'i18next';
+import Helmet from 'react-helmet';
+import { RouteComponentProps } from 'react-router';
+import { useTranslation } from 'react-i18next';
+import {
+  ActionGroup,
+  Button,
+  EmptyState,
+  Title,
+  EmptyStateBody,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { ButtonBar, StatusBox, history, ExternalLink } from '@console/internal/components/utils';
+import { getNamespace, getUID } from '@console/shared';
+import { isTemplateSourceError } from '../../../statuses/template/types';
+import { getTemplateSourceStatus } from '../../../statuses/template/template-source-status';
+import { filterTemplates } from '../../vm-templates/utils';
+import {
+  getTemplateName,
+  getTemplateProvider,
+  isCommonTemplate,
+} from '../../../selectors/vm-template/basic';
+import NamespacedPage, {
+  NamespacedPageVariants,
+} from '../../../../../dev-console/src/components/NamespacedPage';
+import { createVM } from '../../../k8s/requests/vm/create/simple-create';
+import { CreateVMForm } from '../forms/create-vm-form';
+import { getTemplateOSIcon } from '../../vm-templates/os-icons';
+import { formReducer, initFormState } from '../forms/create-vm-form-reducer';
+import { useStorageClassConfigMap } from '../../../hooks/storage-class-config-map';
+import { SUPPORT_URL } from '../../../constants/vm-templates';
+import { useVmTemplatesResources } from '../hooks/use-vm-templates-resources';
+
+const DevConsoleCreateVmFormEmptyState: React.FC<{ templateParam: string; t: TFunction }> = ({
+  templateParam,
+  t,
+}) => (
+  <EmptyState>
+    <Title headingLevel="h4" size="lg">
+      {t('kubevirt-plugin~Error Loading Template')}
+    </Title>
+    <EmptyStateBody>
+      {t('kubevirt-plugin~Virtual machine template')}: <strong>{templateParam}</strong>{' '}
```

Comment:
this should be one translation

```suggestion
      {t('kubevirt-plugin~Virtual machine template {{ templateParam }} not found', { templateParam })}
```